src/info/utils/mod.rs

```diff
@@ -27,16 +27,14 @@ fn to_human_time(time: Time) -> String {
     let since_epoch_duration = SystemTime::now()
         .duration_since(SystemTime::UNIX_EPOCH)
         .unwrap();
-
     let ts = Duration::from_secs(match time.seconds.try_into() {
         Ok(s) => s,
         Err(_) => return "<before UNIX epoch>".into(),
     });
-    let duration = since_epoch_duration.checked_sub(ts).expect(
-        "Achievement unlocked: time travel! \
-        Check your system clock and commit dates.",
-    );
-    let ht = HumanTime::from(-(duration.as_secs() as i64));
+    // Calculate the distance from the current time. This handles
+    // future dates gracefully and will simply return something like `in 5 minutes`
+    let duration = (ts.as_secs() as i64) - (since_epoch_duration.as_secs() as i64);
+    let ht = HumanTime::from(duration);
     ht.to_string()
```

Comment:
Here is a suggestion to simplify the logic and handle commit time before epoch :
```rust
fn to_human_time(time: Time) -> String {
    let since_epoch_duration = SystemTime::now()
        .duration_since(SystemTime::UNIX_EPOCH)
        .expect("System time is before the Unix epoch");

    // Calculate the distance from the current time. This handles
    // future dates gracefully and will simply return something like `in 5 minutes`
    let delta_in_seconds = time.seconds - (since_epoch_duration.as_secs() as i64);
    HumanTime::from(delta_in_seconds).to_string()
}
```
It will also require to update the unit test `display_time_before_epoch` to avoid an underflow